File: CMS/src/app/shared/DanhMucATTPXepLoai.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucATTPXepLoai } from 'src/app/shared/DanhMucATTPXepLoai.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucATTPXepLoaiService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucATTPXepLoai";
    }
}

File: Service/Implement/DanhMucATTPXepLoaiService.cs
namespace Service.Implement
{
    public class DanhMucATTPXepLoaiService : BaseService<DanhMucATTPXepLoai, IDanhMucATTPXepLoaiRepository>
    , IDanhMucATTPXepLoaiService
    {
    private readonly IDanhMucATTPXepLoaiRepository _DanhMucATTPXepLoaiRepository;
    public DanhMucATTPXepLoaiService(IDanhMucATTPXepLoaiRepository DanhMucATTPXepLoaiRepository) : base(DanhMucATTPXepLoaiRepository)
    {
    _DanhMucATTPXepLoaiRepository = DanhMucATTPXepLoaiRepository;
    }
    }
    }

